src/PlainStringsArray.php

```diff
@@ -37,6 +37,16 @@ public function slice(int $offset, int $length): PlainStringsArray
         return new self($this->strings->slice($offset, $length));
     }
 
+    public function skip(int $length): PlainStringsArray
+    {
+        return $this->slice($length, $this->count() - $length);
```

Comment:
I don't remember why `slice` requires `$length`, maybe it shouldn't just like `array_slice`?

Reply:
:thinking: 

would be ok to  set `$length` default value to `PHP_INT_MAX` for `IterableValue`  and `null` for `ArrayValue` ?

Reply:
Not sure, why set an upper limit to IterableValue?

Reply:
i've changed it to null, but honestly i dont believe that anyone ever iterate that much :P

Reply:
Nevet say never :P